### PR TITLE
hera-fuzzer

### LINKED ISSUE #376: Fuzzing Hera
It should be fuzzed with random inputs to be executed (with and without the sentinel).

Most of the fuzzing corpus should be built with the successful test cases of ewasm tests suite. Goal is fuzzing the EEI implementation.

### PR DESCRIPTION
**Pull request by chfast**
Closes #376.

### CODE REVIEW

File: src/CMakeLists.txt [hunk 1]
```diff
@@ -30,8 +30,8 @@ endif()
 
 target_include_directories(hera
     PUBLIC $<BUILD_INTERFACE:${hera_include_dir}>$<INSTALL_INTERFACE:include>
-    PRIVATE ${evmc_include_dir})
-target_link_libraries(hera PRIVATE hera-buildinfo evmc::evmc evmc::instructions binaryen::binaryen Threads::Threads)
+)
+target_link_libraries(hera PUBLIC evmc::evmc PRIVATE hera-buildinfo evmc::instructions binaryen::binaryen Threads::Threads)
```
Comment (axic) [inline src/CMakeLists.txt]: Can we merge this commit in a separate PR already?

**Comment by chfast** [comment]
Ready!

File: test/fuzzing/CMakeLists.txt [hunk 2]
```diff
@@ -0,0 +1,3 @@
+
```
Comment (axic) [inline test/fuzzing/CMakeLists.txt]: I have noticed that every cmake file of yours has a leading empty line. Is there a reason for this? 😉

Comment (chfast) [reply test/fuzzing/CMakeLists.txt]: Not really. I sometimes have license notice here...

**Pull request merged**